WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridFragment.java

```diff
@@ -508,7 +541,22 @@ private void fetchMediaList(boolean loadMore) {
             updateEmptyView(EmptyViewMessageType.LOADING);
             setRefreshing(true);
 
-            FetchMediaListPayload payload = new FetchMediaListPayload(mSite, loadMore);
+            // use a partial mime_type match when fetching media
+            String mimeType;
+            switch (mFilter) {
```

Comment:
I noticed that the network requests aren't including the mime type - the reason is that the cases in this switch are missing `break`s.